strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BrowseStoragesController.java

```diff
@@ -69,4 +87,158 @@ public void browseRepositories(@ApiParam(value = "The storageId", required = tru
 
         getDirectoryListing(storage, request, response);
     }
+    
+    @ApiOperation(value = "Used to browse inside repositories", position = 1)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = ""),
+                            @ApiResponse(code = 404, message = "Requested path not found."),
+                            @ApiResponse(code = 500, message = "Server Error."),
+                            @ApiResponse(code = 503, message = "Repository not in service currently.")})
+    @PreAuthorize("hasAuthority('ARTIFACTS_RESOLVE')")
+    @RequestMapping(value = { "{storageId}/{repositoryId}/{path:.+}" },
+                    method = {RequestMethod.GET},
+                    headers = {"user-agent=Maven/*"})
+    public void browseMavenRepositoriesInternal(@ApiParam(value = "The storageId", required = true)
```

Comment:
> browseMavenRepositoriesInternal
> browseNugetRepositoriesInternal
> browseNpmRepositoriesInternal
> browseRawRepositoriesInternal

are we going to add similar method for each new repository layout ? Can't we just introduce one method with `headers = {"user-agent"}`  (not sure if `headers = {"user-agent"}`   is necessary ?)